spec/graphql/subscriptions_spec.rb

```diff
@@ -119,12 +119,16 @@ class EventSubscription < GraphQL::Schema::Subscription
   end
 
   class Subscription < GraphQL::Schema::Object
-    if TESTING_INTERPRETER
-      extend GraphQL::Subscriptions::SubscriptionRoot
-    else
+    if !TESTING_INTERPRETER
       # Stub methods are required
       [:payload, :event, :my_event].each do |m|
-        define_method(m) { |*a| nil }
+        define_method(m) { |*a|
+          if context.query.subscription_update?
+            object
+          else
+            context.skip
+          end
```

Comment:
I think this was broken all along, but my assertion wasn't good enough to realize it 🤦‍♂